packages/react-components/src/types/Call.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * An object containing information about dominant speakers in the call.
+ * The most dominant speaker is at index 0 in `speakersList` and the second most
+ * dominant is at 1 and so forth.
+ */
+export interface DominantSpeakers {
+  /** A list of speakers who are dominant speakers in a call.
+   * The most dominant speaker is at index 0.
+   * The second dominant at 1 and so on.
+   * */
+  speakersList: ReadonlyArray<string>;
+  /**
+   * Timestamp of when this object was created.
+   */
+  timestamp: Date;
```

Comment:
I'm assuming this is here to potentially avoid too frequent updates?

nit: The comment above is a bit misleading. This is the timestamp when dominant speakers were last updated (not this particular object, which is cached and what not).

Reply:
Right. Will change the language.

Reply:
Infact, removing it for now. You guyz are correct. Logic should reside in bindings.